### ISSUE TITLE
Visualization of the ROIs in a probabilistic atlas

### ISSUE THREAD

**Issue by oesteban** [wiheto/netplotbrain#47]
Probabilistic atlases (e.g., difumo) are not piecewise smooth.

It would be interesting to weigh the colors of the ROIs and modify the alpha channel to make them transparent as you move away from the ROI and enter a different one.

**Comment by wiheto** [comment]
Good idea. Should be easy to add. 

Are "dseg" and "probseg" used for all atlases on TemplateFlow? That would be an easy way to control the behaviour.

**Comment by oesteban** [comment]
I bet @poldrack will be interested in something like this to be easy.

In nilearn they have this: https://nilearn.github.io/stable/auto_examples/01_plotting/plot_prob_atlas.html#sphx-glr-auto-examples-01-plotting-plot-prob-atlas-py but none of the examples seem to make a good job at plotting probabilities.

cc/ @poldrack might be interested in learning about this particular issue and netplotbrain in general.

**Comment by poldrack** [comment]
indeed!  we are moving towards using difumo for many projects so having a way to visualize this would be really helpful

**Comment by wiheto** [comment]
Alright. Will try and add asap then!
